ipn/ipnlocal/expiry.go

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2023 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
Maybe a little file comment up here? (not attached to copyright or package line)